### ISSUE TITLE
Create an "About" page with credits and licenses/attributions

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#673]
### Problem

In order to pay our respects to all of the third-party code/resources we use, and to comply with any licenses that require attribution, we need to put our acknowledgements (and copies of any licenses, as necessary) into an About page.

We don't have an About page.

### Solution

1. Design an About page (cc @mmbotelho)
2. Once designed, implement it and add [attributions for things like SVGs](https://github.com/okTurtles/group-income-simple/pull/671#issuecomment-533232127) and third-party libraries

Because of how simple and straightforward this is, I'm combining the "design" and "implement" aspects of this into a single issue. It will be closed only after both design and implementation are done.

**Comment by sandrina-p** [comment]
This is needed for the prototype, so I added the "Priority:High" label.

**Comment by mmbotelho** [comment]
I have a question about this page - was this issue created specifically because of SVG's from The Noun Project? [See this thread](https://github.com/okTurtles/group-income-simple/pull/671#issuecomment-533232127).

Or are there any others?

Question 2 - About pages are usually accessible from the public website (not the logged in version) because people who don't have an account (or want to learn more about the project) can be interested in this information as well. This is where I would advise keeping the About page in a public version (groupincome.org).

**Comment by taoeffect** [comment]
> Or are there any others?

Our project has a giant list of dependencies. We should acknowledge as many of them as we can on the About page. Think of it like the credits after a movie.

> Question 2 - About pages are usually accessible from the public website (not the logged in version) because people who don't have an account (or want to learn more about the project) can be interested in this information as well. This is where I would advise keeping the About page in a public version (groupincome.org).

I've never seen an About page like that. All of the About pages I've seen are in-app, and ours should be no different.

**Comment by mmbotelho** [comment]
Can you give me an example @taoeffect ? I couldn't find one

**Comment by sandrina-p** [comment]
> Our project has a giant list of dependencies.

Those dependencies usually do not need to be mentioned in the about page. As any open source project, having the link to the repo is enough to show that that (e.g. that we use VueJS and Cypress).

**Comment by taoeffect** [comment]
Per Slack convo, this issue and About page isn't a company or project about page, but rather an app about page. Please open up some random macOS apps about go into the "About [App]" menu to see what what's being referred to.

So, this issue is *not* about these types of about pages (which don't belong in-app, but belong on the website):

- https://www.spotify.com/pt/about-us/contact/
- https://slack.com/intl/en-pt/about?eu_nc=1
- https://about.gitlab.com/company/

It's about these types of "About pages" (really about windows, but I called it a page because that's what we're calling them using our Vue.js nomenclature):

<img width="669" alt="Screen Shot 2020-05-23 at 8 51 43 AM" src="https://user-images.githubusercontent.com/138706/82734921-eb1f8c80-9cd2-11ea-8144-002fc11d9256.png">
<img width="669" alt="Screen Shot 2020-05-23 at 8 51 41 AM" src="https://user-images.githubusercontent.com/138706/82734922-ee1a7d00-9cd2-11ea-8da5-bba76614e8c3.png">
<img width="396" alt="Screen Shot 2020-05-23 at 8 51 22 AM" src="https://user-images.githubusercontent.com/138706/82734923-eeb31380-9cd2-11ea-9640-3a11fdad6f24.png">

Here's an app that combines an About Window with a button that opens an Acknowledgements window:

<img width="462" alt="Screen Shot 2020-05-23 at 9 02 10 AM" src="https://user-images.githubusercontent.com/138706/82735143-59b11a00-9cd4-11ea-8c95-f3baf7aa5127.png">
<img width="672" alt="Screen Shot 2020-05-23 at 9 02 30 AM" src="https://user-images.githubusercontent.com/138706/82735139-56b62980-9cd4-11ea-81f6-af8e4b8c0273.png">

My apologies for the confusion!

**Comment by mmbotelho** [comment]
This is ready to implement. [Figma screens](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=13848%3A20)

Please note that the user settings sidebar also changed slightly, in desktop and mobile screens as well.

**Comment by SebinSong** [comment]
Hi People,
Just noticed that the app doesn't have this page yet. I can pick this up for my next task.
Pls let me know if there's anything I need to note in working on it.

**Comment by taoeffect** [comment]
@SebinSong sounds like a good idea. 👍 

FYI, I think the designs are a bit unfinished, because some of the licenses ask that we include them as part of the app, and the designs Margarida made don't account for they, they just have links to the libraries. So this is something that @leihla can help with in the future, but for now feel free to go ahead and implement them as designed.

**Comment by taoeffect** [comment]
Closed (for the most part) by #1519.

A followup issue can be opened to fill in more acknowledgements, but the page is done! Nice work @Silver-IT!
